### PR TITLE
Updated the metrics for Persistence

### PR DESCRIPTION
**Pull request by guda249**
Updated the Persistence metrics to the modified values in the akka-persistence.json.

Persistence modified metrics  from the application:
1. promexample_mesmer_akka_persistence_event_time_sum
2. promexample_mesmer_akka_persistence_event_time_count
3. promexample_mesmer_akka_persistence_event_time_bucket
4. promexample_mesmer_akka_persistence_recovery_time_sum
5. promexample_mesmer_akka_persistence_recovery_time_count
6. promexample_mesmer_akka_persistence_recovery_time_bucket

### CODE REVIEW

File: example/docker/dashboards/akka-persistence.json [hunk 1]
```diff
@@ -174,7 +174,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum by($persistence_label)(rate(promexample_akka_persistence_recovery_total[1m]))",
+          "expr": "sum by($persistence_label)(rate(promexample_mesmer_akka_persistence_event_time_sum[1m]))",
```
Comment (lgajowy) [inline example/docker/dashboards/akka-persistence.json:177]: (see above)

File: example/docker/dashboards/akka-persistence.json [hunk 2]
```diff
@@ -449,7 +449,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(rate(promexample_akka_persistence_persistent_event_total[$__rate_interval]))",
+          "expr": "sum(rate(promexample_mesmer_akka_persistence_event_time_sum[$__rate_interval]))",
```
Comment (lgajowy) [inline example/docker/dashboards/akka-persistence.json:452]: Similar story as above: `promexample_mesmer_akka_persistence_event_time_count` replaced `promexample_akka_persistence_persistent_event_total`

File: example/docker/dashboards/akka-persistence.json [hunk 3]
```diff
@@ -538,7 +538,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum by($persistence_label)(rate(promexample_akka_persistence_persistent_event_total[$__rate_interval]))",
+          "expr": "sum by($persistence_label)(rate(promexample_mesmer_akka_persistence_event_time_sum[$__rate_interval]))",
```
Comment (lgajowy) [inline example/docker/dashboards/akka-persistence.json:541]: (as above)

**Review by lgajowy — CHANGES_REQUESTED** [review]
Hi. Sorry for the late review - I was on vacation. Thanks for the contribution!

**Comment by lgajowy** [comment]
@guda249 The values presented in the dashboards do not make sense now, unfortunately (events do not take 5 hours for sure):

Screenshot 2022-09-27 at 11 07 53

It could be that I was wrong assuming we can use _sum histogram values. Or maybe some additional division is needed? Do you think you could investigate that?

**Comment by guda249** [comment]
@lgajowy  Sure, will look into this.

**Comment by guda249** [comment]
@lgajowy , I have looked into the metrics as the _sum metrics contains the sum of all time when that particular event has been triggered, couldn't find any proper division, which was giving meaningful graph. Except when i changed the graph timescale from ms to ns.

**Comment by lgajowy** [comment]
@skipper1982 do you think you could take a look at this?

**Comment by skipper1982** [comment]
> @skipper1982 do you think you could take a look at this?

Sure. Will do.

**Comment by skipper1982** [comment]
@guda249: Please change the unit from `ms` to `ns`. And it should change ~5h into ~20ms, which seems about right.

**Comment by guda249** [comment]
Modifed the timeunit from ms to ns

**Review by skipper1982 — APPROVED** [review]
This looks great.

**Review by lgajowy — APPROVED** [review]

**Comment by lgajowy** [comment]
Thanks!

**Pull request merged**